app-misc/gtypist/gtypist-2.9.5-r1.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
```

Comment:
Can we do EAPI=7 and update the rest of the ebuild appropriately?

Reply:
I am not the maintainer of this package and not experienced, so my intention was to just make minimal changes that fix the bug.

Reply:
I reckon this is simple, change to EAPI=7, and possibly be able to drop eutils. See if it still works.

Reply:
Thank you, it was indeed simple.